spot_driver/src/images/spot_image_publisher.cpp

```diff
@@ -90,9 +90,18 @@ bool SpotImagePublisher::initialize() {
   const auto uncompress_images = parameters_->getUncompressImages();
   const auto publish_compressed_images = parameters_->getPublishCompressedImages();
 
+  std::set<spot_ros2::SpotCamera> cameras_used;
+  const auto cameras_used_parameter = parameters_->getCamerasUsed(has_arm_);
+  if (cameras_used_parameter.has_value()) {
+    cameras_used = cameras_used_parameter.value();
+  } else {
+    logger_->logWarn("Invalid cameras_used parameter! Defaulting to publishing from all cameras.");
```

Comment:
@khughes-bdai shall we log what the error message wrapped in the returned `tl::expected` was?